config/25.1/ports.conf

```diff
@@ -14,13 +14,14 @@ databases/p5-DBD-Sybase				arm
 databases/pecl-mongodb				arm
 databases/php${PRODUCT_PHP}-mysqli		arm
 databases/php${PRODUCT_PHP}-sqlite3
-databases/py-duckdb@py${PRODUCT_PYTHON}		arm
+databases/py-duckdb@py${PRODUCT_PYTHON}
 databases/py-redis@py${PRODUCT_PYTHON}		arm
 databases/py-sqlite3@py${PRODUCT_PYTHON}
 databases/pymongo				arm
 databases/redis72				arm
 databases/rrdtool
 devel/arcanist@php${PRODUCT_PHP}		arm
+devel/autoconf
```

Comment:
autoconf is a dependency of automake in the line below, this is sort of implied by the fact that https://pkg.opnsense.org/FreeBSD:14:amd64/25.1/MINT/25.1/latest/All/autoconf-2.72.pkg exists in the packages...

Reply:
interesting! let me see if I can triple check this and attempt to replicate it again as I didn't check to the dependency there and the workaround posted in the PR happened to work and the only package I noticed that was missing was devel/autoconf.  Will see if this acts differently for some reason and report as a bug to freebsd as suggested if so!

Thanks again for that too!

Reply:
in some cases the mere fact that you retry building after a failure can fix problems seen in the FreeBSD ports (and some of these build failures and reorders are probably arch dependant and not seen easily on amd64 which we do nightly builds on)